web/libs/editor/src/components/App/App.jsx

```diff
@@ -47,7 +47,8 @@ import { ToastProvider, ToastViewport } from "@humansignal/ui/lib/toast/toast";
 import { Annotation } from "./Annotation";
 import { BottomBar } from "../BottomBar/BottomBar";
 import Debug from "../Debug";
-import Grid from "./Grid";
+// import Grid from "./Grid";
+import Summary from "../Summary/Summary";
```

Comment:
I would use a more precise name for this component given it is pretty open, maybe `AnnotationSummary` or `ReviewSummary` or anything else that gives a bit more information

Reply:
thank you, good point! renamed to `TaskSummary`